### PR TITLE
chore: Host sample application on Netlify

### PR DESCRIPTION
**Pull request by patricklafrance**
- Host the sample application on Netlify
- Fix a bug with the EventBus where the returned `dispatch` function wasn't bound to the EventBus instance
- Added a FAQ section to the docs

### CODE REVIEW

File: packages/core/src/messaging/useEventBusDispatcher.ts [hunk 1]
```diff
@@ -4,5 +4,7 @@ import { useEventBus } from "../runtime/useEventBus.ts";
 export function useEventBusDispatcher<TEventNames extends EventName>() {
     const eventBus = useEventBus<TEventNames>();
 
-    return eventBus.dispatch;
+    return (eventName: TEventNames, data?: unknown) => {
```
Comment (alexasselin008) [inline packages/core/src/messaging/useEventBusDispatcher.ts:7]: should we wrap this in useCallback?

File: sample/host/src/bootstrap.tsx [hunk 2]
```diff
@@ -2,12 +2,19 @@ import { ConsoleLogger, Runtime, RuntimeContext, registerLocalModules } from "@s
...
 import { createRoot } from "react-dom/client";
 import { register } from "@sample/local-module";
-import { registerRemoteModules } from "@squide/webpack-module-federation";
+import { type RemoteDefinition, registerRemoteModules } from "@squide/webpack-module-federation";
 import { sessionAccessor } from "./session.ts";
 
+const Remotes: RemoteDefinition[] = [
+    {
+        name: "remote1",
+        url: isNetlify ? "https://squide-remote-module.netlify.app" : "http://localhost:8081"
```
Comment (alexasselin008) [inline sample/host/src/bootstrap.tsx:14]: this should be done via a .env file. I feel like we don't want utils methods like "isNetlify" in code that are used as samples.

Comment (patricklafrance) [reply sample/host/src/bootstrap.tsx:14]: I don't share both of your concerns here.

File: packages/core/src/messaging/useEventBusDispatcher.ts (hunk 1 above)
Comment (patricklafrance) [reply packages/core/src/messaging/useEventBusDispatcher.ts:7]: Agreed

**Review by alexasselin008 — APPROVED** [review]

**Pull request merged**